vizro-core/src/vizro/actions/_actions_utils.py

```diff
@@ -113,17 +113,25 @@ def _update_nested_graph_properties(graph_config: Dict[str, Any], dot_separated_
     return graph_config
 
 
-def _get_parametrized_config(targets: List[str], parameters: List[CallbackTriggerDict]) -> Dict[str, Dict[str, Any]]:
+def _get_parametrized_config(
+    targets: List[ModelID], parameters: List[CallbackTriggerDict]
+) -> Dict[ModelID, Dict[str, Any]]:
     parameterized_config = {}
     for target in targets:
         # TODO - avoid calling _captured_callable. Once we have done this we can remove _arguments from
         #  CapturedCallable entirely.
-        graph_config = deepcopy(model_manager[target].figure._arguments)  # type: ignore[index, attr-defined]
+        graph_config = (
+            deepcopy(model_manager[target].figure._arguments)  # type: ignore[attr-defined]
```

Comment:
Ah now I realise these are not `_output_property` but in fact refer to the names of the fields that contain the captured callable.

Shall we have another field in `Graph` and `Table` that sets `_callable_component`?

Reply:
Yes can do, let me try that

Reply:
Done

Reply:
Sorry I wasn't clear here but what I meant was `_callable_component: str = PrivateAttr("figure")` and then here it would use `getattr` to get the relevant field. The `_callable_component` is just telling us what the relevant field is, not a replacement or copy of that field.

Reply:
Ok gotcha, implemented this new version, and used it in the `_page` model to check for attribute